ipynb/Metadata cross correlation.ipynb

```diff
@@ -967,6 +967,72 @@
     "dm = skbio.DistanceMatrix(data=(1 - dm_corr1.values), ids=dm_corr1.index)\n",
     "dm.write('./correlations_dm.txt')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we'll perform higherarchical clustering on the distance matrix, to determine the groups."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "tree_ = skbio.tree.TreeNode.from_linkage_matrix(ward(dm_.condensed_form()), dm_.ids)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We cluster objects according to their correlation distance matrix.\n",
+    "The resulting tree is traversed bottom up: starting at a tip, we go up \n",
+    "until we exceed an accumulated branch length of THRESHOLD.\n",
+    "This node forms the root of a new cluster."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "threshold = 0.29\n",
```

Comment:
can you add: `# chosen as it approximated a cluster width of correlation of 0.5`

and please reword if that isn't as cleanly stated as it should be